subsys/sd/mmc.c

```diff
@@ -9,7 +9,7 @@
 #include <zephyr/sd/mmc.h>
 #include <zephyr/sd/sd.h>
 #include <zephyr/sd/sd_spec.h>
-#include <zephyr/zephyr.h>
+#include <zephyr/kernel.h>
```

Comment:
I think the changes from this commit should be integrated into the commits that introduce these files, rather than having this later commit that makes these updates

Reply:
`zephyr/kernel.h` is currently included twice here, it looks like this might have just gotten messed up during the rebase process